docs/src/test/scala/docs/http/scaladsl/SprayJsonPrettyMarshalSpec.scala

```diff
@@ -43,10 +42,10 @@ class SprayJsonPrettyMarshalSpec extends server.RoutingSpec {
     // verify the pretty printed JSON
     Get("/") ~> service.route ~> check {
       responseAs[String] shouldEqual
-        """{
-          |  "name": "akka",
-          |  "id": 42
-          |}""".stripMargin
+        "{" + "\n" +
+        "  \"name\": \"akka\"," + "\n" +
+        "  \"id\": 42" + "\n" +
+        "}"
```

Comment:
Hmm, this is in the documentation, so we either need to add comments to explain what's going on but it would of course be better to find some less verbose alternative we could offer to Windows users.